driver/normalizer/normalizer.go

```diff
@@ -422,9 +421,27 @@ var Normalizers = []Mapping{
 						{Name: "DeclaresParameterPack", Op: Any()},
 						{Name: "Prop_ArrayModifiers", Op: Any()},
 						{Name: "Prop_Initializer", Optional: "optInitializer", Op: Var("ainit")},
-						{Name: "ExpandedFromMacro", Optional: "optMacro8", Op: Any()},
+						{Name: "ExpandedFromMacro", Optional: "optMacro", Op: Any()},
 						{Name: "Prop_PointerOperators", Optional: "optPointerOps", Op: Any()},
 					},
+					Fields{
+						{Name: uast.KeyType, Op: String("CPPASTElaboratedTypeSpecifier")},
+						{Name: uast.KeyPos, Op: Var("parampos")},
+						{Name: "Prop_Name", Op: Var("aname")},
+						{Name: "Prop_Initializer", Optional: "optInitializer", Op: Var("ainit")},
+						{Name: "Kind", Op: Var("eltype")},
+						{Name: "IsConst", Op: Any()},
```

Comment:
It would be nice to move those nodes to a wrapper node on Type (in a different PR).

Reply:
Yep, the fields were so slightly different between types that it almost wasn't worth it but with these new ones I think it is.